modules/amazon-s3-configuring-acs.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-amazon-s3.adoc
+:_module-type: PROCEDURE
+[id="amazon-s3-configuring-acs_{context}"]
+= Configuring {product-title}
+
+To configure Amazon S3 backups, create a new integration in {product-title}.
+
+.Prerequisites
+* An existing S3 Bucket.
+To create a new bucket with required permissions, see the official Amazon documentation topic link:https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html[How Do I Create an S3 Bucket?]
```

Comment:
When I click the link, the title is just called "Creating a bucket", so I'd use that, and add a period at the end of the sentence.

I think you can also drop "official"

Reply:
They might have changed the title from when I last looked at that link :)

Reply:
Yeah that's what I assumed :)